### ISSUE TITLE
[Accessibility]Inspect: The MSAA structure of NumericUpdown/DomainUpDown in Core is missing some children nodes

### ISSUE THREAD

**Issue by Marie-Shi** [dotnet/winforms#4130]
* .NET Core Version: 
.Net 6.0.100-alpha.1.20516.5

* Have you experienced this same bug with .NET Framework?: 
No

**More info:**
This issue cannot repro on .Net Core 3.1. 

**Minimal repro:**

1. Create a .NET Core project with a Label control and change the text to ‘numericUpDown’.
2. Add a NumericUpDown control to form.
3. Build and run the core project.
4. Open the Inspect tool and observe the NumericUpDown control on MSAA mode.

Or 

1. Extract the “MSAA_Numeric&Domain.zip” and open it in VS.
[MSAA_Numeric&Domain.zip](https://github.com/dotnet/winforms/files/5398625/MSAA_Numeric.Domain.zip)

2. Build and run the core project.
3. Open the Inspect tool and observe the NumericUpDown control on MSAA mode.

**Actual behavior:**
The MSAA structure of NumericUpdown/DomainUpDown in Core is missing some children nodes.
![Core_5 0](https://user-images.githubusercontent.com/45864017/96393447-b5cf8a00-11f1-11eb-806b-1490a52205a8.png)

**Expected behavior:**
The MSAA structure of NumericUpdown/DomainUpDown in Core is correct, it should be like Core 3.1:
![core_3 1](https://user-images.githubusercontent.com/45864017/96393454-ba943e00-11f1-11eb-8bba-92eb7dbfaf4f.png)

**Comment by merriemcgaw** [comment]
MSAA representation is not the focus of current accessibility efforts. If this breaks narrator or something we'll revisit, but for now it's on the backlog and we'll evaluate it in the future

**Comment by merriemcgaw** [comment]
What is the behavior with UIA selected, or in the Accessibility Insights tool?

**Comment by Marie-Shi** [comment]
@merriemcgaw The behavior with UIA mode in inspect and Accessibility Insights tool are all **correct**, please see the below Screenshot.
**UIA mode in inspect:**
![inspect](https://user-images.githubusercontent.com/45864017/98348990-68a64180-2054-11eb-8c3d-7341358e6e96.jpg)
**Accessibility Insights:**
![insight](https://user-images.githubusercontent.com/45864017/98348997-6c39c880-2054-11eb-91f9-91101f5e5a9c.jpg)

**Comment by vladimir-krestov** [comment]
It can be fixed by #4168

**Comment by Marie-Shi** [comment]
Verified this issue with .Net 6.0.100-alpha.1.20562.2 from master branch, it is fixed, now the MSAA structure of NumericUpdown/DomainUpDown in Core is correct.
![Untitled](https://user-images.githubusercontent.com/45864017/99020712-2a4fdb80-259a-11eb-8ecd-2e2a0c31b92e.png)

**Comment by Zheng-Li01** [comment]
Verified this issue in .Net SDK 5.0.101 build, this issue is fixed.
